.eslintignore

```diff
@@ -1 +1,2 @@
 *.node*.js
+node_modules
```

Comment:
I added this cause the linter would yell about `node_modules` files, which you could have in the `/examples` directory.

Reply:
Thank you!